bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -290,7 +290,7 @@ crate::internal_macros::define_extension_trait! {
         ///
         /// [`minimal_non_dust`]: Script::minimal_non_dust
         fn minimal_non_dust_custom(&self, dust_relay_fee: FeeRate) -> Option<Amount> {
-            self.minimal_non_dust_internal(dust_relay_fee.to_sat_per_kwu() * 4)
+            self.minimal_non_dust_internal(dust_relay_fee.to_sat_per_kwu_floor() * 4)
```

Comment:
In 4831689f0181c458dc6acf118414f56fcfe56bd1:

This one should be `_ceil` since we're trying to put a floor on the set of feerates that'll get broadcast.

BTW the commit message says that you add FIXME comments but actually you don't.